src/main/java/org/jenkinsci/plugins/deployment/DeploymentTrigger.java

```diff
@@ -66,14 +73,13 @@ public void checkAndFire(DeploymentFacet facet) {
                         Run b = upstream.getBuildByNumber(n);
                         if (b!=null) {
                             // pass all the current parameters if we can
-                            ParametersAction action = b.getAction(ParametersAction.class);
-                            job.scheduleBuild(job.getQuietPeriod(), new UpstreamDeploymentCause(b), action);
+                            parameterizedJobMixIn.scheduleBuild(new UpstreamDeploymentCause(b));
```

Comment:
@jglick Ouch, I thought just the opposite. Why is `scheduleBuild2` better than `scheduleBuild` in this context?